crates/fuel-streams-core/src/outputs/subjects.rs

```diff
@@ -0,0 +1,161 @@
+use fuel_core_types::{fuel_tx::TxId, fuel_types::ContractId};
+use fuel_streams_macros::subject::{IntoSubject, Subject};
+
+use crate::types::*;
+
+#[derive(Subject, Debug, Clone, Default)]
+#[subject_wildcard = "outputs.>"]
+#[subject_format = "outputs.{tx_id}.{index}.{output_type}"]
```

Comment:
this isn't the way we have defined in the specs

Reply:
![CleanShot 2024-09-12 at 14 59 32@2x](https://github.com/user-attachments/assets/3bcac92c-7930-45d7-9619-10869201c8dd)